.github/workflows/progress-tracker.yml

```
@@ -82,7 +82,7 @@ jobs:
                 Math.min(...nums),
                 Math.max(...nums),
                 (nums.reduce((a,b) => a + b, 0) / nums.length).toFixed(2),
-                (nums.length % 2 ? nums[half] : (nums[half - 1] + nums[half]) / 2.0).toFixed(2)
+                nums[half]
```

Why did you change the logic of median calculation? Was anything wrong in it? I had some testing locally before adding the code I did. Removing `.toFixed(2)` and surrounding parenthesis was all that could have been removed.

```suggestion
                nums.length % 2 ? nums[half] : (nums[half - 1] + nums[half]) / 2.0
```

---

Commented in https://github.com/HTTPArchive/almanac.httparchive.org/pull/1060#discussion_r456934285